CalibTracker/SiStripHitEfficiency/BuildFile.xml

```diff
@@ -24,5 +24,6 @@
 <use   name="TrackingTools/PatternTools"/>
 <use   name="TrackingTools/TrackFitters"/>
 <use   name="TrackingTools/MeasurementDet"/>
+<use   name="TrackingTools/KalmanUpdators"/>
```

Comment:
For my education, why do you need to add TrackingTools/KalmanUpdators?

Reply:
Because it's included here and in the line after:
https://github.com/cms-sw/cmssw/blob/master/CalibTracker/SiStripHitEfficiency/interface/HitEff.h#L14
Before, it somehow got this dependency indirectly by one of the used RecoTracker packages, but since the dependencies got cleaned there this didn't happen anymore. That's why we have to declare the dependency at this right place now.